server/server.go

```
@@ -72,9 +149,94 @@ func (n *Node) Start() {
+		log.Println("Server could not stop gracefully in time. Performing force shutdown.")
+		n.grpcServer.Stop()
+		data["message"] = "Server stopped forcefully"
+	})
+	defer timer.Stop()
+
+	n.RaftNode.Shutdown()
+	n.grpcServer.GracefulStop()
+	log.Println("Server stopped gracefully.")
+	data["message"] = "Server stopped gracefully"
+
+	json.NewEncoder(w).Encode(data)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)